<?xml version="1.0"?>
<launch>
    <node name="ai_ball" pkg="web_cam" type="image_streaming.py" output="screen">
        <remap from="web_cam_img" to="ai_ball_img"/>
        <remap from="web_cam_settings_service" to="front_cam_settings"/>
        <param name="ip_address" value="192.168.54.162"/>
        <param name="overlay_text" value="AI-ball"/>
    </node>
</launch>
